docs/api/doc.md

```diff
@@ -0,0 +1,470 @@
+
+
+`snapshot` -- [Snapshot]({% link api/snapshot.md %})
+
+> The snapshot to ingest
+
+{: .warn }
+> The snapshot **must** include its [`data`]({% link api/snapshot.md %}#data--object), [`v`]({% link api/snapshot.md %}#v--number) and [`type`]({% link api/snapshot.md %}#type--type) properties
+
+{: .d-inline-block }
+
+`callback` -- Function
+
+Optional
+{: .label .label-grey }
+
+> ```js
+> function(error) { ... }
+> ```
+
+> A callback that will be called once the snapshot has been fetched
+
+### destroy()
+
+Unsubscribe and stop firing events. Also removes the document reference from its [`Connection`]({% link api/connection.md %}), allowing the `Doc` to be garbage-collected.
+
+```js
+doc.destroy([callback])
+```
+
+{: .d-inline-block }
+
+`callback` -- Function
+
+Optional
+{: .label .label-grey }
+
+> ```js
+> function(error) { ... }
+> ```
+
+> A callback that will be called when the document has been destroyed
+
+### create()
+
+Create the document locally and send the create operation to the server.
+
+```js
+doc.create(data [, type [, options [, callback]]])
+```
+
+`data` -- Object
+
+> The document contents. The structure will depend on the document's [type]({% link types/index.md %})
+
+`type` -- [Type]({% link types/index.md %})
+
+> The document's [type]({% link types/index.md %})
+
+{: .d-inline-block }
+
+`options` -- Object
+
+Optional
+{: .label .label-grey }
+
+> Default: `{}`
+
+> `options.source` -- any (optional)
+
+> > Default: `true`
+
+> > {{ page.copy.submit_source }}
+
+{: .d-inline-block }
+
+`callback` -- Function
+
+Optional
+{: .label .label-grey }
+
+> ```js
+> function(error) { ... }
+> ```
+
+> A callback that will be called when the document has been committed by the server
+
+### submitOp()
+
+Apply an operation to the document locally and send the operation to the server. The document must have been fetched or subscribed to
+
+```js
+doc.submitOp(op [, options [, callback]])
+```
+
+`op` -- Object
+
+> The op to submit. The structure of `op` depends on the document's [type]({% link types/index.md %})
+
+{: .d-inline-block }
+
+`options` -- Object
+
+Optional
+{: .label .label-grey }
+
+> Default: `{}`
+
+> `options.source` -- any (optional)
+
+> > Default: `true`
+
+> > {{ page.copy.submit_source }}
+
+{: .d-inline-block }
+
+`callback` -- Function
+
+Optional
+{: .label .label-grey }
+
+> ```js
+> function(error) { ... }
+> ```
+
+> A callback that will be called when the op has been committed by the server
+
+### del()
+
+Delete the document locally and send a delete operation to the server.
+
+```js
+doc.del([options [, callback]])
+```
+
+{: .info }
+ShareDB documents and their ops are never truly deleted from the database. Instead, they will be tombstoned.
```

Comment:
Maybe we can explain what a tombstone looks like for us 🤔 
Here is a suggestion but I'm not sure I'm capturing all of it. 

```suggestion
ShareDB documents and their ops are never truly deleted from the database. Instead, they will be tombstoned. A tombstone is a document that has no data, but retains some metadata such as the original `_id`. ShareDB will ignore these tombstones when queried, but they are still present in the underlying database. 
```